.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Isch das echt es File wo du is .gitignore intuechenntsch? Oder muess das im version control dinne si?

Reply:
ja hans jetzt is gitignore ta